src/admin/src/components/composite/advisorySummaryView/AdvisorySummaryView.js

```diff
@@ -182,11 +110,20 @@ export default function AdvisorySummaryView({
               {!p.url && p.protectedAreaName}
             </div>
           ))}
+          {advisory.protectedAreas.length > 5 &&
+            <button
+              type="button"
+              className="btn btn-link btn-boolean d-block my-2"
+              onClick={() => setShowParks(!showParks)}
+            >
+              {showParks ? "Hide" : "Show"} all parks affected
+            </button>
+          }
           <Chip
             icon={<FileCopyOutlinedIcon />}
-            label="Copy all"
+            label="Copy all URLs"
             clickable
-            className="ad-copy"
+            className="ad-copy bcgov-button bcgov-normal-white"
             onClick={() => {
```

Comment:
This button doesn't copy the urls to my clipboard

Reply:
I think this was already broken.  Maybe just create a new ticket.  

`navigator.clipboard.writeText()` seems to work better, but I still don't understand the purpose of passing the URLs to the clipboard as HTML